requirements.txt

```diff
@@ -1,5 +1,6 @@
 matplotlib>=3.1.3
 numpy>=1.18.1
 pandas>=1.0.1
+requests
```

Comment:
As directly imported ideally should list `requests` as a dependency (though it will probably be pulled in by one of the other dependencies here).